### ISSUE TITLE
Hotkey not working

### ISSUE THREAD

**Issue by BourgeoisDirk** [Belphemur/SoundSwitch#1041]
### What happened?

Been using this gem of a program for a while, but i migrated to a laptop, installed it, configured it.
And the Hotkey only works when I set it up again. If i put the PC to Sleep/Hibernate/Restart/Shutdown the Hotkey no longer works.

### Step to reproduce

Install program, configure with 2 audio outputs and a Hotkey.
Set the program to start with Windows.
Sleep/Hibernate/Restart/Shutdown
When the PC is logged in again, the App is running, but the Hotkey no longer works.

It's set to "Alt + Control + F11", i have to change it to something else, then back to "Alt + Control + F11"
After doing so, the hotkey works again.

Here is a clip that after Startup, showing shortcut not doing anything, after changing it, and changing it back, works again:
https://user-images.githubusercontent.com/51524225/198819362-c684b412-2405-4f38-bed4-5a09f093a459.mp4

### Version

6.4.2.0

### What version of Windows are you seeing the problem on?

Windows 10

**Comment by NeedHelp00** [comment]
I'm having the exact same issue.  Laptop on Windows 10, hooked up to a thunderbolt 3 docking station. Was using the SoundSwitch app to change audio from headphones to my soundbar. (Headphones fro gaming and then soundbar for everything else).  Looks like when my computer reboots or wakes up from sleep mode, the functionality of the "Control + Alt + F11" no longer works.  App is great when it works, but lately, no bueno.  Please fix.  Thanks!

**Comment by thebro** [comment]
I am having the simmilar issue on Windows 11 as well.
Hotkey stops working after a while.
I was able now to get it working simply by unchecking "Hotkey enabled" and then checking it again.
But I remember also having to change the hotkey combination to something else and then back to "Alt + Control + F11", but not sure if it was in this or some previous version.
Tried now to reproduce the issue by putting PC into hibernation and then starting it up again, but the hotkey worked for me after starting the PC. So still not sure why it stops working for me...
